modules/hivemq-edge-module-opcua/src/main/java/com/hivemq/edge/adapters/opcua/OpcUaAdapterConfig.java

```diff
@@ -34,6 +34,13 @@ public class OpcUaAdapterConfig extends AbstractProtocolAdapterConfig {
                        required = true)
     private @NotNull String uri;
 
+    @JsonProperty("overrideUri")
+    @ModuleConfigField(title = "Override server returned endpoint URI",
+                       description = "Overrides the endpoint URI returned from the OPC-UA server with the hostname and port from the specified URI.",
+                       format = ModuleConfigField.FieldType.BOOLEAN,
+                       required = false)
+    private @NotNull Boolean overrideUri = true;
```

Comment:
sorry, missed this in the initial review, but the default should be `false` here, as the mechanism to respect the server returned URI is intended by OPC UA

Reply:
```suggestion
    private @NotNull Boolean overrideUri = false;
```